public/js/components/ObjectInspector.js

```diff
@@ -131,7 +131,7 @@ const ObjectInspector = React.createClass({
     let objectValue;
     if (nodeHasProperties(item) || nodeIsPrimitive(item)) {
       const object = item.contents.value;
-      objectValue = Rep({ object });
+      objectValue = Rep({ object, mode: "tiny" });
```

Comment:
Thanks @wldcordeiro. I'm okay moving forward with this and adjusting if we need to.